test/server/cmd/grpc_client.go

```diff
@@ -19,31 +19,33 @@ var grpcClientLog = grpcLog.WithName("client")
 func newGRPCClientCmd() *cobra.Command {
 	args := struct {
 		address string
+		unary   bool
 	}{}
 	cmd := &cobra.Command{
 		Use:   "client",
 		Short: "Run GRPC Test Server",
 		Long:  "Run GRPC Test Server.",
 		Example: `
-# Start a GRPC client that connects to localhost:8080, opens stream and 
-# sends "Request #${n_of_streams}.${n_of_requests}" every 2 seconds.
-test-server grpc client address="localhost:8080"`,
+# Start a GRPC client that connects to localhost:8080, and
+# sends "Request #${n_of_streams}.${n_of_requests}" every 1 seconds.
+test-server grpc client address="localhost:8080" unary=true`,
```

Comment:
Without an option to do unary requests there was only 1 connection and no load balancing took place so I couldn't assert on errors.